packages/hawtio/src/plugins/camel/TreeProcessor.ts

```diff
@@ -0,0 +1,105 @@
+import React from 'react'
```

Comment:
Code styling: I would reserve a capitalised file name for React components. We could change it to `treeProcessor.ts` or `tree-processor.ts`.